settings/i18n/pt-BR.po

```diff
@@ -0,0 +1,22 @@
+msgid "appname"
+msgstr "Fit-Boy"
+
+
+
+msgid "thm"
+msgstr "Tema"
+
+msgid "clr"
+msgstr "Cor"
+
+msgid "concurstop"
+msgstr "Parar a animação do cursor do console"
+
+msgid "bgop"
+msgstr "Opacidade do plano de fundo"
+
+msgid "dwlr"
+msgstr "Morador"
```

Comment:
Dweller's translation to portuguese could either be "Morador" or "Habitante" (similar to Spanish) but I decided to as the Fallout Wiki Fandom suggests: https://fallout.fandom.com/pt-br/wiki/Vault_Dweller